packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue

```diff
@@ -75,4 +76,17 @@ li {
 label:not(:last-child)::after {
 	content: ', ';
 }
+
+.truncate {
+	max-width: 180px;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
```

Comment:
Food for thought: names tend to get pretty long in scientific community - I think we will need a better way of dealing with them on top of truncations.

Reply:
Agree. I was hoping to use my "truncate after 3 lines" trick, but it changes the positions of the ports below and the connection links aren't aware of the new port locations. A one line truncate is better than before, where the text spilled out of the container! Maybe we can do this one together at the hackathon?

Reply:
Sure, lets chat in the hackathon.